### PR TITLE
For #185 "Keep query in search field"

### PR DESCRIPTION
**Pull request by LiamGoodacre**

### CODE REVIEW

File: src/Foundation.hs [hunk 1]
```diff
@@ -106,6 +106,7 @@ instance Yesod App where
 
         manalytics <- appAnalytics . appSettings <$> getYesod
         isSearch <- testCurrentRoute (== SearchR)
+        searchText <- map (maybe "" unpack) (lookupGetParam "q")
```
Comment (hdgarrood) [inline src/Foundation.hs]: Would `fromMaybe ""` work here? I think the templates ought to work the same way when splicing `Text` values in as opposed to `String` values.

Comment (LiamGoodacre) [inline src/Foundation.hs]: Yep, good call.

**Comment by hdgarrood** [comment]
:+1: thank you!

**Comment by LiamGoodacre** [comment]
Thanks!

**Pull request merged**